shared/src/api/interceptors.coffee

```diff
@@ -0,0 +1,134 @@
+_ = require 'lodash'
+minimatch = require 'minimatch'
+{Promise} = require 'es6-promise'
+
+makeLocalRequest = (requestConfig) ->
+  {url} = requestConfig
+
+  [uri, params] = url.split('?')
+  if requestConfig.method is 'GET'
+    url_parts = ["#{uri}.json", params]
+  else
+    url_parts = ["#{uri}/#{requestConfig.method}.json", params]
+    requestConfig.mockMethod = requestConfig.method
+    requestConfig.method = 'GET'
+
+  requestConfig.url = _.compact(url_parts).join('?')
+
+makeLocalResponse = (response) ->
+  payload = if response.config.data then JSON.parse(response.config.data) else {}
+  response.data = _.extend({}, response.data, payload)
+
+doesErrorMatch = (handledErrors, errorName) ->
+  _.indexOf(handledErrors, errorName) > -1 or
+    _.some(handledErrors, _.partial(minimatch, errorName))
```

Comment:
oh thanks! i always have a hard time finding/remember `_.includes` cause i try to look for like `_.contain` or `_.has` or something and then i give up and do the `_.indexOf` thing.